client/src/components/Home.js

```diff
@@ -9,54 +10,77 @@ const clickArt = (artId, history) => {
   history.push('/auction/' + artId);
 };
 
+const onImageClick = (event, history) => {
+  var split = event.target.src.split('?');
+  var lastIndex = split.length - 1;
+  var lastCharacter = Number(split[lastIndex]);
+  clickArt(lastCharacter, history);
+}
+
 //render the description ... as floating right of the image
 //closed auctions should be rendered differently from the ongoing ones
-let MainArt = ({art, history}) => {
-  console.log('heyhey:' , art.first_name);
-  return (
-    <span>
-      <Image className="ui image" src={art.artwork.image_url} onClick={() => {
-        clickArt(art.artwork.id, history);
-      }} />
-      <span className="ui label">
-        <span>Artist: {art.first_name} {art.last_name}</span>
-        <br />
-        <span> Description: {art.artwork.description}</span>
-        <br />
-        <span> Year: {art.artwork.age}</span>
-        <br />
-        <span> Closing price: missing!!!!</span>
-      </span>
-    </span>
-  );
-}
+// let MainArt = ({art, history}) => {
+//   return (
+//     <Grid.Column>
+//       <Image src={art.artwork.image_url}/>
+//       <Card.Content>
+//         <Card.Header>{art.first_name} {art.last_name} </Card.Header>
+//         <Card.Meta>{art.artwork.age}</Card.Meta>
+//         <Card.Description>{art.artwork.description}</Card.Description>
+//       </Card.Content>
+//     </Grid.Column>
+//   );
+// }
 
 let MainArts = ({mainArts, history}) => {
-  console.log('mainarts: ', mainArts);
   if (!mainArts[0]) {
     return <p>loading~~</p>
   } else {
-    // this className in div is not working:
+    let images = []
+    mainArts.forEach((item) => {
+      let imageObj = {
+        original: `${item.artwork.image_url}?${item.artwork.id}`,
+        description: `${item.first_name} ${item.last_name} Closing Price: $${item.buyout_price}`, 
+      }
+      images.push(imageObj);
+    })
     return (
-      <Container className="ui small images">
-       {mainArts.map(mainArt => <MainArt key={mainArt.id} art={mainArt} history={history}/>)}
-      </Container>
+      <div style={{'max-height': '400px', 
```

Comment:
Is there a way for us to move the css into a css file instead of inline in the code?